content/en/dashboards/ddsql_editor/reference/_index.md

```diff
@@ -8,7 +8,7 @@ DDSQL is in private beta.
 
 ## Overview
 
-DDSQL is a query language for Datadog data. It implements several standard SQL functions, such as `SELECT`, and allows queries against unstructured data, such as [tags][1].
+DDSQL is a query language for Datadog data. It implements several standard SQL operations, such as `SELECT`, and allows queries against unstructured data, such as [tags][1].
```

Comment:
`SELECT` is a statement, not a function; in SQL, a function is of the form `function_name(args, …)`. "operation" is a more general term encompassing both.